util/install.sh

```diff
@@ -102,6 +148,15 @@ case $(uname) in
         #elif [[ -e /etc/redhat-release ]]; then
         #    # CentOS/RedHat
         #    os=RedHat
+        else # unrecognized Linux distro
+            os=
+            # try to grab a Linux distro identifier
+            {
+                set -o pipefail
+                hint=$(lsb_release -i | cut -f2) ||
```

Comment:
Will there be a problem if `lsb_release` or `/etc/redhat-release` does not exist?